app/models/user.rb

```diff
@@ -51,6 +51,7 @@ class User < ActiveRecord::Base
 
   before_validation :default_display_name, on: [:create, :update]
   before_validation :setup_unsubscribe_token, on: [:create]
+  before_validation :update_ouroboros_created, on: [:create]
```

Comment:
Yep - looks like it'll get called [here](https://github.com/plataformatec/devise/blob/181920886e4e60783f330b94072b0423345f714f/lib/devise/strategies/database_authenticatable.rb#L11)